db/migrations/1723751676020-Data.js

```diff
@@ -0,0 +1,33 @@
+module.exports = class Data1723751676020 {
+    name = 'Data1723751676020'
+
+    async up(db) {
+        await db.query(`DROP INDEX "public"."IDX_36524a30939bb522f1eda95267"`)
+        await db.query(`DROP INDEX "public"."IDX_b05770c7b9f6fba9a0b12814e9"`)
+        await db.query(`ALTER TABLE "send_packet_fee_deposited" DROP COLUMN "send_gas_limit"`)
+        await db.query(`ALTER TABLE "send_packet_fee_deposited" DROP COLUMN "send_gas_price"`)
+        await db.query(`ALTER TABLE "send_packet_fee_deposited" ADD "recv_gas_limit" numeric NOT NULL`)
+        await db.query(`ALTER TABLE "send_packet_fee_deposited" ADD "recv_gas_price" numeric NOT NULL`)
```

Comment:
That won't work as is because there are already records in the table.

Reply:
What's the correct way to do this?

Reply:
There are various ways to solve this:
1) Truncate `send_packet_fee_deposited` table. If you want to backfill from the beginniner, you also need to truncate the table that tracks the progress of the processor.
2) Allow NULL values 
3) Set a default value 
4) Instead of dropping the column and adding a new one, you just rename it (the best in my mind)

Reply:
Doing 2 and 4

Reply:
@Inkvi Went with #4, let me know if that looks right

Reply:
Were you able to run the migration locally?

Reply:
Yup

Reply:
`sqd migration:apply` wouldn't work if the file wasn't right?

Reply:
Correct